src/axom/core/utilities/nvtx/interface.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2017-2020, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+
+#include "axom/core/utilities/nvtx/interface.hpp"
+
+namespace axom
+{
+namespace nvtx
+{
+  
+/*!
+ * \brief Internal data-structure to hold the NVTX settings.
+ */
+static struct settings_t
+{
+  uint32_t color;     /*!< the associated color with an NVTX Range */
```

Comment:
Please change these to ``axom::uint32``, here and elsewhere.

Reply:
This one, I need to think a bit about. 

The only reason we added the axom aliases in the first place was to support fixed width types when compiled without C++11. Since, we now require C++11, these are probably not needed anymore. So, there is no way that `axom::uint32` and `uint32_t` would be a different type, in which case, my preference is to use these types directly. Less indirection.

I'll create a Github issue on this.